README-en.md

```diff
@@ -0,0 +1,217 @@
+  encoding processing, compression, encryption and decryption, configuration, HTTP client, etc., which are convenient to
+  get started developing business code quickly
+- Provides a flexible expansion mechanism. Easy to support multiple protocols, service discovery, load balancing,
+  monitoring and alerting, [call tracking](flare/doc/), etc.
+- Extensive optimizations for modern architectures. For example, the [scheduling group](flare/) and
+  [object pool](flare/doc/object-pool.md) of [NUMA aware](https://en.wikipedia.org/wiki/Non-uniform_memory_access),
+  [zero-copy buffer](flare /doc/buffer.md), etc.
+- High-quality code. Strict adherence to [Google C++ Code Specification](https://google.github.io/styleguide/cppguide.html),
+  with 80% test coverage
+- Complete [documentation](flare/doc) and [examples](flare/example) and [debugging support](flare/doc/debugging.md),
+  easy to get started quickly
+
+## System Requirements
+
+- Linux 3.10 kernel and above, no other OS supported at this time
+- x86-64 processors, aarch64 and ppc64le are also supported, but have not been used in production environments
+- GCC 8 and above compiler
+
+## Getting Started
+
+Flare comes out of the box with the required [third-party libraries](thirdparty/), so there is usually no need to install additional dependent libraries. Just pull the code and use it under Linux.
+
+The tarball under `thirdparty/` we store via [Git LFS](https://git-lfs.github.com/), so you need to make sure that `git-lfs` is properly installed before pulling the code.
+
+### Building
+
+We use [`blade`](https://github.com/chen3feng/blade-build) to build this project.
+
+- Compile: `./blade build ...`.
+- Test: `./blade test ...`.
+
+After that, you can refer to the introduction in [Getting Started Guide](flare/doc/intro-rpc.md) to build a simple RPC service.
+
+### Debugging
+
+We believe that the [debugging](flare/doc/debugging.md) experience is also an important part of the development and maintenance process, and we have made some support for this as follows.
+
+- [GDB plugin for listing pending fibers](flare/doc/gdb-plugin.md)
+- [Support for various Sanitizers](flare/doc/sanitizers.md)
+
+### Testing
+
+To improve the experience of writing a single test, we provide a number of tools for [writing single tests](flare/doc/testing.md).
+
+These include, but are not limited to.
+
+- [RPC Mock](flare/testing/rpc_mock.h)
+- [Redis Mock](flare/testing/redis_mock.h)
+- [COS Mock](flare/testing/cos_mock.h)
+- [HTTP Mock](flare/testing/http_mock.h)
+- [Non-virtual Function Mock](flare/testing/hooking_mock.h)
+- Some tool methods, etc.
+
+Translated with www.DeepL.com/Translator (free version)
+
+### Example
+
+We provided some [examples](flare/example) for reference, the following code snippet is a simple relay service (include both client and service).
+
+```cpp
+#include "thirdparty/gflags/gflags.h"
```

Comment:
We've removed all `thirdparty/` prefixes in our code (so as to support other building systems), unfortunately I forgot to remove it from the sample code. 

Mind to remove the prefix from both `README.md` and `README-en.md`?